.github/trivy_scan.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Can we move this script to a common place? May be `infra-utils` repo and we get it from there ?

Reply:
Yes @mohan-13. 👍